tests/trestle/core/commands/author/ssp_test.py

```diff
@@ -369,6 +369,42 @@ def test_ssp_assemble(tmp_trestle_dir: pathlib.Path) -> None:
     # confirm the file was not written out since no change
     assert orig_ssp_path.stat().st_mtime == orig_file_creation
 
+    # Tests the removal of component definitions that are no longer valid for an ssp
+    orig_ssp, orig_ssp_path = ModelUtils.load_model_for_class(tmp_trestle_dir, ssp_name, ossp.SystemSecurityPlan)
+    imp_reqs = orig_ssp.control_implementation.implemented_requirements
+    components = orig_ssp.system_implementation.components
+
+    new_component = gens.generate_sample_model(ossp.SystemComponent)
+    new_component.uuid = '46b7a556-72bb-4281-b805-a8f4030ca0e3'
+    new_component.title = 'foo'
```

Comment:
please define this uuid string once and refer to it as a constant rather than duplicating the string